scripts/makefilegen.py

```diff
@@ -91,6 +91,7 @@
 argparser.add_argument('--mainclockperiod', help='Clock period of default clock, in nanoseconds', type=int, default=10)
 argparser.add_argument('--derivedclockperiod', help='Clock period of derivedClock, in nanoseconds', type=float, default=5.0)
 argparser.add_argument('--pcieclockperiod', help='Clock period of PCIE clock, in nanoseconds', type=int, default=None)
```

Comment:
let's rename "xsimflags" to "simflags" and use that to pass arguments to all simulators.